src/runtime.rs

```diff
@@ -26,18 +30,64 @@ extern "system" {
     ) -> u32;
 }
 
-#[link(name = "windowsapp")]
-extern "system" {
-    pub fn CoIncrementMTAUsage(cookie: *mut RawPtr) -> ErrorCode;
+unsafe fn load_proc(library_name: &str, sym: &str) -> Result<RawPtr, ErrorCode> {
```

Comment:
Can we document why this is unsafe?

Reply:
Hmm, it isn't actually unsafe according to rust rules, AFAICT, it's just unsafe contents being called by unsafe code and I was lazy. I can fix it.

Reply:
Thanks! FYI: Unsafe functions in the future will require unsafe parts to be marked as unsafe. This will be a warning soon, and a hard error in a future edition of Rust.